pkg/memio/ioport_linux_amd64.s

```diff
@@ -0,0 +1,60 @@
+// Copyright 2012-2020 the u-root Authors. All rights reserved
```

Comment:
Is this any different from the 386.s file?

Reply:
yes, stack offsets differ.